packages/walletkit/src/contract-utils.ts

```diff
@@ -13,9 +13,9 @@ import { getGasPriceMinimumContract } from './contracts'
 import { getGoldTokenAddress } from './erc20-utils'
 import { Logger } from './logger'
 
-const gasInflateFactor = 1.3
+const gasInflateFactor = 1.5
```

Comment:
Why changing this?

Reply:
because Baklava

Reply:
What changed in Baklava?

Reply:
No idea but I was seeing failures due to underestimated gas amounts